script.js

```diff
@@ -114,21 +114,18 @@ $(document).ready(function(){
     myLoop();
   }
 
-  //functions
+  //helper functions
   var strictModeToggle = function(button){
     if(strictMode){
```

Comment:
when i first read this `strictMode` I was confused because javascript has a thing called 'use strict'. Maybe `gameStrictMode` ?